src/components/Typography/HX/__snapshots__/HX.test.tsx.snap

```diff
@@ -4,7 +4,7 @@ exports[`HX Component renders H1 correctly 1`] = `
 <DocumentFragment>
   <h1
     aria-label="Text"
-    class="ant-typography"
+    class="mia-platform-typography"
```

Comment:
will this prefix be used only in these tests or is thios classs used in production build as well? tbh I'd prefer using this class also in producion build

Reply:
It is already used within components (also for production build 😉 )